static/app/views/projectEventRedirect.tsx

```diff
@@ -50,10 +50,14 @@ function ProjectEventRedirect({router}: Props) {
       // responseURL is the URL of the document the browser ultimately loaded,
       // after following any redirects. It _should_ be the page we're trying
       // to reach; use the router to go there.
-
+      //
+      // Here we assume that the responseURL is on the same domain as the
+      // as the window.location.origin, so we simply take the pathname.
+      //
       // Use `replace` so that hitting the browser back button will skip all
       // this redirect business.
-      router.replace(xhr.responseURL);
+      const url = new URL(xhr.responseURL);
+      router.replace(url.pathname);
```

Comment:
This loses search parameters, are we OK with that?

Reply:
Good point, it looks like it is possible there are query params and query hash in the responseURL I'll add it back

Reply:
nvm, looking at the backend some more, it's guaranteed to only not contain query params or query hash.